src/ddos.sh

```diff
@@ -16,6 +16,7 @@ CONF_PATH="${CONF_PATH}/"
 
 # Other variables
 BANS_IP_LIST="/var/lib/ddos/bans.list"
+SERVER_IP_LIST=`ifconfig | awk -F "[: ]+" '/inet addr:/ { print $4 }' | xargs | sed -e 's/ /|/g'`
```

Comment:
This doesn't works with newer versions of ifconfig as found on archlinux, the following one does works with debian versions of ifconfig and newer ones as that found on archlinux:

```sh
ifconfig | grep "inet " | awk '{print $2}' | sed "s/addr://g" | xargs | sed -e 's/ /|/g'
```